tests/regression/37-congruence/01-simple.c

```diff
@@ -0,0 +1,34 @@
+//PARAM: --enable ana.int.congruence  --enable ana.int.congruence_no_overflow --disable ana.int.def_exc --disable ana.int.enums
+
+int main() {
+	int a = 1;
+	int b = 2;
+	int c = 3;
+	int d = 4;
+	int e = 0;
+
+	while (d < 9)	 {
+		b = 2 * a;
+		d = d + 4;
+		e = e - 4 * a;
+
+		a = b - a;
+		c = e + d;
+	}
+
+	a = d / 2;
+	b = d % 2;
+
+    // c is unknown
+	assert (c == 4); // UNKNOWN!
```

Comment:
`//UNKNOWN!` has the following meaning:
- The analysis claiming that this must fail is unsound
- The analysis claiming that this must succeed is unsound

Or to phrase it differently, this means that **in the concrete** there are both executions where this holds and where it doesn't. In this case, `//UNKNOWN` is the correct annotation, as this holds in the concrete and it just the analysis that can not show it.

Reply:
I think in the remaining tests, you used it correctly.